bot_impl.py

```diff
@@ -506,10 +506,11 @@ async def call_next(self):
         toCall = self.order[self.position]
         player_banshee_ability = the_ability(toCall.character, Banshee)
         player_is_active_banshee = player_banshee_ability and player_banshee_ability.is_screaming
+        voudon_is_active = next((x for x in global_vars.game.seatingOrder if isinstance(x.character, Voudon) and not x.is_ghost), None)
```

Comment:
Do you maybe want to use this instead to make it clear that we don't actually care about the returned user?  I had to look up the next( ..., None) syntax becuase it took me a minute to figure out why we care about the first element of a list (since really we're just caring if the list is empty or not)

```py
voudon_is_active = any(isinstance(x.character, Voudon) and not x.is_ghost for x in global_vars.game.seatingOrder)
```